src/pods/test-sentences/test-sentences.mappers.ts

```diff
@@ -0,0 +1,21 @@
+import * as sentenceApi from 'core/sentences';
+import * as sentenceVm from './test.sentences.vm';
+import * as verbApi from 'core/verbs/global-verbs.api';
+import { splitSentence } from './test-sentences.business';
+
+export const mapFromSentenceApiToSentenceVm = (
```

Comment:
For this scenario is a good idea to remove the second parameter and directly search in the mapper for the verb tenses

Reply:
done